ansible/playbooks/adhoc/uninstall.yml

```diff
@@ -113,26 +122,34 @@
         - "{{ bin_dir }}/kubelet"
         - "{{ bin_dir }}/kube-proxy"
         - "{{ bin_dir }}/kube-scheduler"
+        - "{{ bin_dir }}/flanneld"
+        - /run/flannel_docker_opts.env
+        - /run/flannel
+        - /tmp/flannel-conf.json
+        - /opt/flannel-0.5.5 #flannel version can change
         - /etc/kubernetes
         - /etc/systemd/system/kube-addons.service
         - /etc/systemd/system/kube-controller-manager.service
         - /etc/systemd/system/kube-proxy.service
+        - /etc/systemd/system/kube-proxy.service.wants
         - /etc/systemd/system/kube-apiserver.service
+        - /etc/systemd/system/kube-apiserver.service.wants
         - /etc/systemd/system/kubelet.service
         - /etc/systemd/system/kube-scheduler.service
+        - /etc/systemd/system/flanneld.service
+        - /etc/systemd/system/flanneld.service.wants
+        - /etc/systemd/system/multi-user.target.wants
```

Comment:
Since ansible is not managing the following:

kube-proxy.service.wants
kube-apiserver.service.wants
flanneld.service.wants
multi-user.target.wants
delay-master-services.target
delay-node-services.target

Can you try removing and see if uninstall works as expected? I prefer not removing things we don't config.